functionary/ui/templates/core/environment_detail.html

```diff
@@ -27,6 +27,25 @@ <h1 class="title is-1">
                     <i class="fa fa-users"></i>&nbsp;{{ environment.team.name }}
                 </div>
             </div>
+
+            <div class="field">
+                {% if not environment_create_perm %}
+                <label class="label" for="users">
```

Comment:
The way that you access the environment page now is something of a placeholder.  Eventually the entire page should be locked behind the ability to change things, since its only purpose is the administration of the environment.  So don't worry about disabling specific elements of the page.  It should eventually be the case that if you can get to the page, then you can do all of the things on it.